src/FileService/Startup.cs

```diff
@@ -146,9 +149,19 @@ public void Configure(IApplicationBuilder app)
                     .AllowAnyHeader()
                     .AllowAnyMethod());
 
+            var rabbitMqConfig = Configuration
+                .GetSection(BaseRabbitMqOptions.RabbitMqSectionName)
+                .Get<RabbitMqConfig>();
+
             app.UseEndpoints(endpoints =>
             {
                 endpoints.MapControllers();
+
+                endpoints.MapHealthChecks($"/{rabbitMqConfig.Password}/h�", new HealthCheckOptions

```

Comment:
WTF is this -> **hñ** ?????

Reply:
I don't know how it works. Yesterday we fixed it, why it here again it is secret. Fixed